prime-router/settings/STLTs/NJ/nj-dph.yml

```diff
@@ -0,0 +1,122 @@
+      suppressHl7Fields: null
+      suppressAoe: false
+      defaultAoeToUnknown: false
+      replaceUnicodeWithAscii: false
+      useBlankInsteadOfUnknown: null
+      truncateHDNamespaceIds: true
+      truncateHl7Fields: "OBX-23-1,ORC-12-3,OBR-16-3,OBX-17-1,ORC-12-1,OBR-16-1"
+      usePid14ForPatientEmail: false
+      convertTimestampToDateTime: null
+      cliaForOutOfStateTesting: null
+      cliaForSender: {}
+      phoneNumberFormatting: "STANDARD"
+      suppressNonNPI: false
+      processingModeCode: null
+      replaceDiiWithOid: null
+      applyOTCDefault: false
+      useOrderingFacilityName: "STANDARD"
+      valueSetOverrides: {}
+      nameFormat: "STANDARD"
+      receivingOrganization: null
+      convertPositiveDateTimeOffsetToNegative: false
+      stripInvalidCharsRegex: null
+      convertDateTimesToReceiverLocalTime: false
+      useHighPrecisionHeaderDateTimeFormat: false
+      type: "HL7"
+      truncationConfig:
+        truncateHDNamespaceIds: true
+        truncateHl7Fields:
+        - "OBR-16-1"
+        - "OBR-16-3"
+        - "OBX-17-1"
+        - "OBX-23-1"
+        - "ORC-12-1"
+        - "ORC-12-3"
+        customLengthHl7Fields: {}
+    jurisdictionalFilter:
+    - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state.exists()\
+      \ and Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state\
+      \ = 'NJ') or (Bundle.entry.resource.ofType(Patient).address.state.exists() and\
+      \ Bundle.entry.resource.ofType(Patient).address.state = 'NJ')"
+    qualityFilter:
+    - "Bundle.entry.resource.ofType(DiagnosticReport).result.resolve().where(method.empty()\
+      \ or value.coding.code.empty()).count() = 0"
+    - "Bundle.entry.resource.ofType(DiagnosticReport).where(basedOn.resolve().extension('https://reportstream.cdc.gov/fhir/StructureDefinition/effective-date').empty()\
+      \ and specimen.resolve().collection.collected.empty() and effective.empty()\
+      \ and issued.empty() and result.resolve().effective.empty()).count() = 0"
+    - "Bundle.entry.resource.ofType(Organization).identifier.where(type.coding.code\
+      \ = 'CLIA' or system = 'CLIA').where(value.getIdType() != 'CLIA').count() =\
+      \ 0"
+    - "Bundle.entry.resource.ofType(Patient).where(address.line[0].empty() and address.postalCode.empty()\
+      \ and telecom.where(system = 'phone').value.empty() and telecom.where(system\
+      \ = 'email').value.empty()).count() = 0"
+    - "Bundle.entry.resource.ofType(Patient).where(name.family.empty() or name.given.empty()\
+      \ or birthDate.empty()).count() = 0"
+    - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+    - "Bundle.identifier.value.empty().not()"
+    routingFilter:
+    - "Bundle.entry.resource.ofType(MessageHeader).extension('https://reportstream.cdc.gov/fhir/StructureDefinition/sender-id').where(value\
```

Comment:
This also needs to include Simple-report

Reply:
It is SimpleReport